modules/SerialCommHub/manifest.yaml

```diff
@@ -36,6 +36,12 @@ provides:
         description: GPIO direction, false means low for TX, true means high for TX
         type: boolean
         default: false
+      max_packet_size:
+        description: Maximum size of a packet to read/write in bytes. Payload exceeding the size will be chunked.
```

Comment:
You state in your MR description that the limits are < 253, why are you using a default of 256 and why do you allow a maximum of 65k? I think the limits from wikipedia should be in the description and you should explain the max and mins too.

Reply:
well I understand it that the pdu is 253 - the apu (which is what we send --> pdu with check sum and address)  is 256 - so I've defaulted to it. Maybe we should rename the variable to max_adu_size? Or we let the user specify the maximum payload size and treat the checksum as implementation detail - I'm fine with both. 

Regarding the max - I reasoned that there might be devices which can accept larger packets...

Reply:
Your explanation makes sense, but it should be here in a comment somewhere.